.editorconfig

```diff
@@ -1,15 +1,15 @@
-# Editorconfig for consistent coding styles
-# http://editorconfig.org
+root = true
 
 [*]
+charset = utf-8
 end_of_line = lf
 insert_final_newline = true
 indent_style = space
-indent_size = 2
-
-[*.php]
-charset = utf-8
 indent_size = 4
+trim_trailing_whitespace = true
 
-[*.{json,yml}]
```

Comment:
What about indentation in the composer.json file? That we want 2 spaces for json files.

Reply:
Changed to 2 spaces for json files.